### DISCUSSION TITLE
help with forecasting

### DISCUSSION THREAD

**Discussion by steplp** [ourownstory/neural_prophet#730] [Q&A - get help using NeuralProphet]
when NeuralProphet use default parameters, it can forecast the future data, but it not accurate, the forecast curve is soo smooth, but the orignal curve is not like this, so I set n_lags, but result is it just can forecast the original datas, but not can forecast the furture datas, when I set n_forecasts, then will happen memery error! I don't how to do, please help me!

**Comment by yaosting** [comment]
You can use **n_ forecasts** parameter
